### PR TITLE
Improve Airzone integration documentation

### PR DESCRIPTION
**Pull request by Noltari**
## Proposed change
Improve Airzone integration documentation.

## Type of change
- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/airzone.markdown [hunk 1]
```diff
@@ -18,13 +18,13 @@ ha_codeowners:
 ha_integration_type: integration
 ---
 
-This integration allows getting values from the local API of Airzone HVAC zoning systems.
+This integration interacts with the Local API of [Airzone HVAC zoning systems](https://www.airzone.es/en/).\
+**Please note that not all Airzone devices have a Local API.**\
```
Comment (frenck) [inline source/_integrations/airzone.markdown]: I find this a bit weird to be the second sentence in an integration introduction?

File: source/_integrations/airzone.markdown [hunk 2]
```diff
@@ -18,13 +18,13 @@ ha_codeowners:
 ha_integration_type: integration
 ---
 
-This integration allows getting values from the local API of Airzone HVAC zoning systems.
+This integration interacts with the Local API of [Airzone HVAC zoning systems](https://www.airzone.es/en/).\
```
Comment (frenck) [inline source/_integrations/airzone.markdown]: Please don't use the `\`. Thanks!

File: source/_integrations/airzone.markdown [hunk 3]
```diff
@@ -18,13 +18,13 @@ ha_codeowners:
...
+**Please note that not all Airzone devices have a Local API.**\
+Some devices can only be controlled via the Airzone Cloud API and the oldest ones can only be controlled via Modbus protocol.
```
Comment (frenck) [inline source/_integrations/airzone.markdown]: Cool? Now what? As in, what should I do?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/airzone.markdown (hunk 2 above)
Comment (Noltari) [reply source/_integrations/airzone.markdown]: Replaced with double whitespace in ac00452aae956e7b5c512e5752da84675ab84cea

File: source/_integrations/airzone.markdown (hunk 1 above)
Comment (Noltari) [reply source/_integrations/airzone.markdown]: Removed this in ac00452aae956e7b5c512e5752da84675ab84cea

File: source/_integrations/airzone.markdown (hunk 3 above)
Comment (Noltari) [reply source/_integrations/airzone.markdown]: Removed in ac00452aae956e7b5c512e5752da84675ab84cea

**Review by frenck — COMMENTED** [review]
This looks really odd on some devices:

CleanShot 2022-04-12 at 18 56 26

Hard enters/new lines can look weird/different depending on the device the website is viewed at. In general, it's better not to use them.

Either combine a couple of those sentences or make use of a new line with paragraphs. Adding a bit of space generally improves readability.

**Comment by Noltari** [comment]
> This looks really odd on some devices:
> 
> CleanShot 2022-04-12 at 18 56 26
> 
> Hard enters/new lines can look weird/different depending on the device the website is viewed at. In general, it's better not to use them.
> 
> Either combine a couple of those sentences or make use of a new line with paragraphs. Adding a bit of space generally improves readability.

Fixed in 1cdfb42d4af6649cea8893ebcba322f7e92fd224

**Review by frenck — APPROVED** [review]
Thanks, @Noltari 👍

**Pull request merged**